### PR TITLE
Лабораторная работа №4 Кулемин Павел (перезалив)

### PR DESCRIPTION
**Pull request by George-Amberson**
@ggladilov, sorry for add u commit in old pr, i fix it.

### CODE REVIEW

File: llvm/lib/Target/AArch64/AArch64ISelLowering.cpp [hunk 1]
```diff
@@ -4700,33 +4700,65 @@ SDValue AArch64TargetLowering::LowerMGATHER(SDValue Op,
                                MGT->getMemOperand(), IndexType, ExtType);
   }
 
+  // Lower fixed length gather to a scalable equivalent.
```
Comment (ggladilov) [inline llvm/lib/Target/AArch64/AArch64ISelLowering.cpp:4703]: These changes should not be here

File: llvm/lib/Transforms/PeepholeOptimizationCourse/ReplaceInt8AddNewPassManager/ReplaceInt8SumNewPassManager.cpp [hunk 2]
```diff
@@ -0,0 +1,31 @@
+#include "llvm/Transforms/PeepholeOptimizationCourse/ReplaceInt8SumNewPassManager.h"
```
Comment (ggladilov) [inline llvm/lib/Transforms/PeepholeOptimizationCourse/ReplaceInt8AddNewPassManager/ReplaceInt8SumNewPassManager.cpp:1]: Did you dupliate sources for passes in include and Transforms folders?

**Review by ggladilov — COMMENTED** [review]
Do not see commands you used to get .ll files

File: llvm/lib/Target/AArch64/AArch64ISelLowering.cpp (hunk 1 above)
Comment (George-Amberson) [reply llvm/lib/Target/AArch64/AArch64ISelLowering.cpp:4703]: yes, sorry thats my fault. I guess when i did git cherry-pick, this code added into my commit

File: llvm/lib/Transforms/PeepholeOptimizationCourse/ReplaceInt8AddNewPassManager/ReplaceInt8SumNewPassManager.cpp (hunk 2 above)
Comment (George-Amberson) [reply llvm/lib/Transforms/PeepholeOptimizationCourse/ReplaceInt8AddNewPassManager/ReplaceInt8SumNewPassManager.cpp:1]: I’m sorry, it wasn’t done intentionally, I’ll fix it with the new commit

**Comment by ggladilov** [comment]
Fixes did not meet the deadline

**Pull request closed without merging**